### ISSUE TITLE
Enhancement: add note splashes

### ISSUE THREAD

**Issue by NotTheCoderGb** [KadeArchive/Kade-Engine#1241]

**Comment by theDetourist** [comment]
[No](https://github.com/KadeDev/Kade-Engine/pull/308).

**Comment by AmberHaxe** [comment]
It can't be added to K.E until week 8 releases cause of ninjamuffin

although I do believe someone here made a PR for it, so if you really want note splashes just find that P.R, I wouldn't recommend it though.

**Comment by arceuss** [comment]
wouldnt that break the wares rule on gamebanana
just be patient and wait for week 7 on desktop

**Comment by Jaduplex** [comment]
We can't do this until week 7 becomes open source due to ninjamuffin's wishes.

**Comment by RagdollClash** [comment]
Requesting a close.

**Comment by isakube** [comment]
why are we classifying this as a week 7 feature

**Comment by AmberHaxe** [comment]
> wouldnt that break the wares rule on gamebanana
> just be patient and wait for week 7 on desktop

not necessarily, it would go against ninjamuffin's wishes but I've seen a few mods on G.B with note splashes

ex. agoti

**Comment by Jaduplex** [comment]
> > wouldnt that break the wares rule on gamebanana
> > just be patient and wait for week 7 on desktop
> 
> not necessarily, it would go against ninjamuffin's wishes but I've seen a few mods on G.B with note splashes
> 
> ex. agoti

hm, weird.

**Comment by arceuss** [comment]
> 
> 
> > wouldnt that break the wares rule on gamebanana
> > just be patient and wait for week 7 on desktop
> 
> not necessarily, it would go against ninjamuffin's wishes but I've seen a few mods on G.B with note splashes
> 
> ex. agoti

yeah but agoti is based off micd up which i think still has the splashes

**Comment by flowerbearttv** [comment]
> > > wouldnt that break the wares rule on gamebanana
> > > just be patient and wait for week 7 on desktop
> > 
> > 
> > not necessarily, it would go against ninjamuffin's wishes but I've seen a few mods on G.B with note splashes
> > ex. agoti
> 
> yeah but agoti is based off micd up which i think still has the splashes

yeah but mic'd up has a gb page i think, still weird
